podvm/Dockerfile.podvm_binaries.rhel

```diff
@@ -17,7 +17,11 @@ ARG AA_KBC="offline_fs_kbc"
 ARG ARCH
 ARG CAA_SRC
 ARG CAA_SRC_REF
+ARG IMAGE_URL="/tmp/rhel.qcow2"
+ARG IMAGE_CHECKSUM
```

Comment:
I don't think it's needed in the binaries dockerfile, only in the podvm one, and it's already being set there, no?